src/frontend/web_application/src/scenes/Discussion/components/MessagePi/style.scss

```diff
@@ -76,21 +86,30 @@
   }
 
   &__metaphor {
-    padding: 0.825rem 0;
-    border-top: 2px solid #999999;
-    border-bottom: 2px solid #999999;
     color: #999999;
-    font-size: 0.625rem;
+    font-size: 0.825rem;
     font-style: italic;
   }
+
+  @include breakpoint(medium) {
+    &__illustration, &__meter, &__numeric {
+      width: 100%;
+    }
+
+    &__metaphor {
+      padding: 0.825rem 0;
+      border-bottom: 2px solid #999999;
+      font-size: 0.625rem;
```

Comment:
As discussed before summer, we should leave the responsibility of the font size to the body: https://github.com/CaliOpen/Caliopen/pull/1015

Reply:
According to UI mock-up, this should be smaller than body font, and is relative to it, not sure about what you mean here ? Too small ?